### DISCUSSION TITLE
Disabling bound parameters when executing queries

### DISCUSSION THREAD

**Discussion by bryannho** [sqlalchemy/sqlalchemy#10922] [Usage Questions]
I am using `sqlalchemy-snowflake` to execute a query which includes JSON:

```python
%sql select parse_json('{"fruit":"banana"}'):fruit
```

`:fruit` is valid Snowflake SQL, [see here](https://docs.snowflake.com/en/sql-reference/functions/get_path).

Executing this statement provides this error:
```
(sqlalchemy.exc.InvalidRequestError) A value is required for bind parameter 'fruit'
[SQL: select parse_json('{"fruit":"banana"}')%(fruit)s]
```

SQLAlchemy mistakes `:fruit` for a bound parameter. How might I execute this query with **bound parameters disabled** / ignoring instances of `:x`? 

The flow of my code makes escaping the colon with `\` not an option.
I would prefer an alternative to `connection.execute()` which avoids parsing of bound parameters. I have used `connection.exec_driver_sql()` but I still require the other compilation/transpilation/rollback steps that `.exec_driver_sql` misses.

**Comment by CaselIT** [comment]
Hi,

You can use exec_driver_sql, otherwise you will need to escape the colon

**Reply by bryannho** [reply]
@CaselIT Are there any other options other than `exec_driver_sql`? This misses other compilation steps that I need (transpilation, rollback, etc.). Is there any way to modify `.execute` to avoid bound parameters?

I am not able to escape the colon either with the current design of my system.

**Reply by CaselIT** [reply]
Nothing that's not equivalent to using exec_driver_sql
